src/go/rpk/pkg/cli/security/role/create.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package role
+
+import (
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/adminapi"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func createCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	return &cobra.Command{
+		Use:   "create [ROLE]",
+		Short: "Create a role in Redpanda",
+		Long: `Create a role in Redpanda.
+
+After creating a role you may bind ACLs to the role using the '--allow-role' 
+flag in the 'rpk security acl create' command.`,
+		Args: cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			f := p.Formatter
+			if h, ok := f.Help([]string{}); ok {
```

Comment:
Here and other []string{} -- It may be worth changing this to a simple struct with a single field for the array. It's not clear what `[]string` means, while `{role:[]string}` would be clearer

Also why is this an array here?